### PR TITLE
triple identifier pattern

### PR DESCRIPTION
**Pull request by AaronGoldman**

### CODE REVIEW

**Comment by mikestaub** [comment]
Could this scheme also be used for privacy preservation? I could generate a unique triple for all of my contacts and only allow them to email me at that ID. That way if I get spam I know exactly who shared my info and can hold them accountable.

File: TOPICS/identity/example_identifiers.md [hunk 1]
```diff
@@ -9,3 +9,30 @@
 golda~bluesky-community.net
 
 acct:bob@example.com
+
+mailto:alice@example.com
+
+```
+triple identifyer pattern
+@ . Adresses (names given by an authority)
```
Comment (gvelez17) [inline TOPICS/identity/example_identifiers.md:17]: Addresses

File: TOPICS/identity/example_identifiers.md [hunk 2]
```diff
@@ -9,3 +9,30 @@
...
+
+combined types
+username1@sub_domain2.~jzzuedxe2pudvujm#bob
+   Here ~jzzuedxe2pudvujm is identified by hash
+   ~jzzuedxe2pudvujm authorised a subdomain named sub_domain2
+   sub_domain2.~jzzuedxe2pudvujm has authorised a user username1
+   username1@sub_domain2.~jzzuedxe2pudvujm has the display name bob
+   if bob has his own keys they can bind them
+   (username1@sub_domain2.~jzzuedxe2pudvujm ~3oy3p4ucysnf5xee #bob)
+   if ~3oy3p4ucysnf5xee wants to move to a new domain the retract one asertion and isue a new one
```
Comment (wesc) [inline TOPICS/identity/example_identifiers.md:35]: typos: "then retract one assertion and issue a new one"

File: TOPICS/identity/example_identifiers.md [hunk 3]
```diff
@@ -9,3 +9,30 @@
...
 acct:bob@example.com
+
+mailto:alice@example.com
+
+```
+triple identifyer pattern
+@ . Adresses (names given by an authority)
+    . re-delegable authority
+    @ terminal authority
+#   hashtags (names that are not unique)
```
Comment (wesc) [inline TOPICS/identity/example_identifiers.md:20]: How do we handle spaces in the display name? What about unicode, etc?

**Comment by wesc** [comment]
Would it be useful to add a little more plain English narrative around how this would work?

**Review by gvelez17 — APPROVED** [review]

**Pull request merged**